e2e/apprunner/apprunner_test.go

```diff
@@ -165,6 +165,26 @@ var _ = Describe("App Runner", Ordered, func() {
 			}
 		})
 
+		It("should return correct secrets", func() {
+			fmt.Printf("\n\nsecrets: %+v\n\n", svc.Secrets)
+			expectedSecrets := map[string]string{
+				"my-ssm-param": "e2e-apprunner-ssm-param",
+			}
+			for _, secret := range svc.Secrets {
+				Expect(secret.Value).To(Equal(expectedSecrets[secret.Name]))
+			}
+		})
+
+		It("should return correct secrets for backendSvc", func() {
+			fmt.Printf("\n\nsecrets: %+v\n\n", backendSvc.Secrets)
+			expectedSecrets := map[string]string{
+				"USER_CREDS": "arn:aws:secretsmanager:us-east-1:067305307211:secret:e2e-apprunner-MyTestSecret",
```

Comment:
We should just compare if the show result secret ARN contains `e2e-apprunner-MyTestSecret`, since the region and account could be different depending on your local environment.

Reply:
True, I thought so too! Will do that.

Reply:
Done